src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -283,6 +284,8 @@ async function createCluster(previousData: any, context: azure.Context): Promise
         } };
     const createResult = await azure.createCluster(context, options);
 
+    reporter.sendTelemetryEvent("createCluster", { result: createResult.result.succeeded ? "success" : "fail", clusterType: previousData.clusterType });
```

Comment:
Elsewhere we have all-lowercased event names e.g. `cloudselection`, and we've used nouns rather than verbs.  Suggest `clustercreation` perhaps, or `clustercreated` or something like that?